app/components/progressive_image/progressive_image.js

```diff
@@ -110,25 +106,33 @@ export default class ProgressiveImage extends PureComponent {
             inputRange: [20, 100],
             outputRange: [0.2, 1],
         });
-        const defaultOpacity = this.state.intensity.interpolate({
-            inputRange: [0, 100],
-            outputRange: [0.5, 0],
-        });
 
-        const containerStyle = {
-            backgroundColor: changeOpacity(theme.centerChannelColor, defaultOpacity),
-        };
+        const iconColor = changeOpacity(theme.centerChannelColor, 0.32);
+        let iconSize;
+        let containerStyle;
+        if (isSmallImage) {
+            iconSize = 24;
+            containerStyle = {
+                width: iconSize,
+                height: iconSize,
+            };
+        } else {
+            containerStyle = {backgroundColor: changeOpacity(theme.centerChannelColor, 0.08)};
+        }
 
         return (
             <Animated.View style={[styles.defaultImageContainer, style, containerStyle]}>
-                <DefaultComponent
-                    resizeMode={resizeMode}
-                    resizeMethod={resizeMethod}
-                    onError={onError}
-                    source={thumb}
-                    style={[imageStyle, {tintColor: theme.centerChannelColor, opacity: defaultOpacity}]}
+                {!this.state.imageLoaded &&
+                <FileAttachmentIcon
+                    defaultImage={true}
+                    smallImage={isSmallImage}
+                    iconColor={iconColor}
+                    iconSize={iconSize}
+                    backgroundColor={backgroundColor}
+                    theme={theme}
```

Comment:
this should be the thumbnail from the image, soon to use the base64 image included in the metadata if  available, please revert